mix.exs

```diff
@@ -42,6 +45,10 @@ defmodule LiveViewCounter.MixProject do
 
       # LiveView
       {:phoenix_live_view, "~> 0.10.0"},
+      {:floki, ">= 0.0.0", only: :test},
```

Comment:
https://hex.pm/packages/floki

Reply:
Yeah, the official LiveView instructions include `floki` for testing. 👍